cli/flox/doc/flox-build.md

```diff
@@ -21,9 +21,9 @@ flox [<general-options>] build
 # DESCRIPTION
 
 Build the specified `<package>` from the environment in `<path>`,
-and output build artifacts at `result-<package>` adjacent to the environment.
+and output build packages at `result-<package>` adjacent to the environment.
```

Comment:
(non-blocking, including other instances of the same phrase)

There's something about mixing "build packages" as a verb-noun and noun-noun that feels odd.
```suggestion
and output built packages at `result-<package>` adjacent to the environment.
```